src/components/SEO.js

```diff
@@ -49,6 +50,14 @@ function SEO({ description, lang, meta, title, type }) {
 					name: 'twitter:card',
 					content: 'summary'
 				},
+				{
+					name: 'og:image',
```

Comment:
```suggestion
					property: 'og:image',
```

The [Open Graph spec](https://ogp.me/) says it should be `property=`, while Twitter uses `name=`.

Reply:
> Is there something we can do to ensure that src/images/event/hack-banner.png and static/image/hack-banner.png stay in sync in the future? We don't want to accidentally update one but not the other.
> 
> Also, is there a SVG version of the banner available?

"We can get an SVG, but why do we need it?" - Jody

Reply:
> "We can get an SVG, but why do we need it?" - Jody

Good to always have a source file in case we need to modify it (or generate higher resolution PNGs) in the future.

Reply:
We have it in figma which we can extract the svg from it.